### PR TITLE
#1650 standard cr page

### LINKED ISSUE #1650: Change Request standard details page redesign
### Current Features
branch off feature/Change-Request-Redesign for feature branch

image

### Desired Additional Features

We want to take out the page blocks and make the information look nicer. This is for the standard change request types.

### Screenshots (as needed)

image
Epic https://github.com/Northeastern-Electric-Racing/FinishLine/issues/1645
branch off Change-Request-Redesign for feature branch
Message Zach Marino in slack for any design questions!

### PR DESCRIPTION
**Pull request by RChandler234**
## Changes

Fleshed out details pages for Change Requests. Still need to fix the left side of the standard CR page cause it's all spread out rn. Also need to fix the light mode proposed solutions background color.

## Screenshots

image
image
image
image
image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #1650  (issue #)

### CODE REVIEW

File: src/frontend/src/layouts/PageTitle/PageTitle.tsx [hunk 1]
```diff
@@ -40,12 +41,13 @@ const PageTitle: React.FC<PageTitleProps> = ({ title, previousPages, headerRight
         bgcolor={theme.palette.background.default}
       >
         <Grid container>
-          <Grid item xs={6} md={4}>
+          <Grid item xs={6} md={tabs ? 4 : 8} display="flex" alignItems={'center'}>
```
Comment (walker-sean) [inline src/frontend/src/layouts/PageTitle/PageTitle.tsx:44]: On small (especially mobile) screens, the title becomes so stretched that it's one letter per line

**Review by walker-sean — COMMENTED** [review]
Things look really good so far! What sticks out to me is that the non-standard CR page seems a bit visually confusing (it almost seems like the details belong to the same columns because of the way theyre arranged). I like the spacing between the first two rows, and giving the same spacing between the second and third row could make things look cleaner.

**Comment by RChandler234** [comment]
image

**Comment by RChandler234** [comment]
image
image
image
image

**Review by walker-sean — APPROVED** [review]
Looks good!

**Pull request merged**